### PR TITLE
Polish English writing in Chapter 2

### PR DESCRIPTION
**Pull request by wangkuiyi**

### CODE REVIEW

File: 02.recognize_digits/README.md [hunk 1]
```diff
@@ -1,22 +1,20 @@
 # Recognize Digits
 
-The source code for this tutorial is live at [book/recognize_digits](https://github.com/PaddlePaddle/book/tree/develop/02.recognize_digits). For instructions on getting started with Paddle, please refer to [installation instructions](https://github.com/PaddlePaddle/book/blob/develop/README.md#running-the-book).
+The source code for this tutorial lives at [book/recognize_digits](https://github.com/PaddlePaddle/book/tree/develop/02.recognize_digits). For instructions on getting started with Paddle, please refer to [installation instructions](https://github.com/PaddlePaddle/book/blob/develop/README.md#running-the-book).
```
Comment (putcn) [inline 02.recognize_digits/README.md]: shall we change "lives" to "locates"?

File: 02.recognize_digits/README.md [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
+When one learns to program, the first task is usually to write a program that prints "Hello World!". In Machine Learning or Deep Learning, a similar task is to train a model to recognize hand-written digits using the dataset [MNIST](http://yann.lecun.com/exdb/mnist/). Handwriting recognition is a classic image classification problem. The problem is relatively easy and MNIST is a complete dataset. As a simple Computer Vision dataset, MNIST contains images of handwritten digits and their corresponding labels (Fig. 1). The input image is a $28\times28$ matrix, and the label is one of the digits from $0$ to $9$. All images are normalized, meaning that they are both rescaled and centered.
```
Comment (abhinavarora) [inline 02.recognize_digits/README.md]: Lets replace `a similar task ...` to  `an equivalent task ...`.

File: 02.recognize_digits/README.md [hunk 3]
```diff
@@ -1,22 +1,20 @@
...
+The source code for this tutorial lives at [book/recognize_digits](https://github.com/PaddlePaddle/book/tree/develop/02.recognize_digits). For instructions on getting started with Paddle, please refer to [installation instructions](https://github.com/PaddlePaddle/book/blob/develop/README.md#running-the-book).
 
 ## Introduction
-When one learns to program, the first task is usually to write a program that prints "Hello World!". In Machine Learning or Deep Learning, the equivalent task is to train a model to recognize hand-written digits on the dataset [MNIST](http://yann.lecun.com/exdb/mnist/). Handwriting recognition is a classic image classification problem. The problem is relatively easy and MNIST is a complete dataset. As a simple Computer Vision dataset, MNIST contains images of handwritten digits and their corresponding labels (Fig. 1). The input image is a $28\times28$ matrix, and the label is one of the digits from $0$ to $9$. All images are normalized, meaning that they are both rescaled and centered.
```
Comment (abhinavarora) [inline 02.recognize_digits/README.md]: It should be `on the MNIST dataset` instead of `on the dataset MNIST`

File: 02.recognize_digits/README.md [hunk 4]
```diff
@@ -1,22 +1,20 @@
...
+MNIST has been the tester of many image recognition technologies, including single layer linear classifier, Multilayer Perceptron (MLP) and Multilayer CNN LeNet\[[1](#references)\], K-Nearest Neighbors (k-NN) \[[2](#references)\], Support Vector Machine (SVM) \[[3](#references)\], Neural Networks \[[4-7](#references)\], Boosting \[[8](#references)\] and preprocessing methods like distortion removal, noise removal, and blurring.  Among these algorithms, the *Convolutional Neural Network* (CNN) has achieved a series of impressive results in Image Classification tasks, including VGGNet, GoogLeNet, and ResNet (See [Image Classification](https://github.com/PaddlePaddle/book/tree/develop/03.image_classification) tutorial).
```
Comment (abhinavarora) [inline 02.recognize_digits/README.md]: Lets begin the sentence as `The MNIST dataset has been used for evaluating many image recognition algorithms such as a single layer classifier, Multilayer Perceptron ...`

**Comment by abhinavarora** [comment]
Also run the pre-commit for converting markdown to html.

File: 02.recognize_digits/README.md (hunk 1 above)
Comment (wangkuiyi) [reply 02.recognize_digits/README.md]: Done

File: 02.recognize_digits/README.md (hunk 2 above)
Comment (wangkuiyi) [reply 02.recognize_digits/README.md]: Done

File: 02.recognize_digits/README.md (hunk 4 above)
Comment (wangkuiyi) [reply 02.recognize_digits/README.md]: Done

File: 02.recognize_digits/README.md (hunk 2 above)
Comment (wangkuiyi) [reply 02.recognize_digits/README.md]: done

File: 02.recognize_digits/README.md (hunk 3 above)
Comment (wangkuiyi) [reply 02.recognize_digits/README.md]: done

File: 02.recognize_digits/README.md (hunk 1 above)
Comment (wangkuiyi) [reply 02.recognize_digits/README.md]: done

**Review by putcn — APPROVED** [review]
LGTM

**Pull request merged**